db/migrate/20220707204226_add_user_id_post_id_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserIdPostIdToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_column :likes, :user_id, :integer
+    add_column :likes, :post_id, :integer
+  end
+end
```

Comment:
- [ ] The same thing here. Kindly use this `rails g migration addReferencesToLike post:references user:references `, it will generate a migration like this 

```.rb
class AddReferencesToLike < ActiveRecord::Migration[7.0]
  def change
    add_reference :likes, :post, null: false, foreign_key: true
    add_reference :likes, :user, null: false, foreign_key: true
  end
end

```